newm-server/src/main/kotlin/io/newm/server/config/repo/ConfigRepository.kt

```diff
@@ -47,5 +47,6 @@ interface ConfigRepository {
         const val CONFIG_KEY_EMAIL_WHITELIST = "email.whitelist"
         const val CONFIG_KEY_DISTRIBUTION_PRICE_USD = "distribution.price.usd"
         const val CONFIG_KEY_OUTLET_STATUS_CHECK_MINUTES = "outlet.statusCheckMinutes"
+        const val CONFIG_KEY_RECAPTCHA_ENABLED = "recaptcha.enabled" // TODO: remove this flag after both web & mobile clients support recaptcha
```

Comment:
Remove the TODO. I like the idea of leaving it configurable in case we have some issue in production where we need to temporarily disable it to allow someone to get some music through.